WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> Look at our localisation file.

It breaks your retarded golden rule. You said that code should be easily discernable at first glance. Sunni Berber Company for the Valentianian Holy Order breaks your stupid golden rule. You offered to add a pound sign comment, like I did for "DONAMONAPRISC_DECADENT", which also breaks your stupid golden rule.

Your golden rule only applies to me, not you. That is why you are a hypocrite.